.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -4,45 +4,39 @@ about: Request a new feature
 labels: feature-request, status/needs-triage
 ---
 
-<!-- NOTE:📕 If issue regarding developer guide, please create issue [here](https://github.com/awsdocs/aws-cdk-guide/issues) -->
+<!-- short description of the feature you are proposing: -->
+
+
+
+
+### Use Case
+
+<!-- why do you need this feature? -->
 
-## :rocket: Feature Request
 
-### General Information
-<!--
-Check the box below (with an X) if you are able and willing to propose an
-implementation for the requested feature. This does not imply a commitment from
-you to actually do it!
--->
-* [ ] :wave: I may be able to implement this feature request
-<!--
-Check the box below (with an X) if you think this feature might result in a
-breaking change (this requiring a major version bump when released). If unsure,
-please leave the box un-checked.
--->
-* [ ] :warning: This feature might incur a breaking change
 
-### Description
```

Comment:
If we are cutting down to one description section, I think "Description" makes more sense than "Proposed Solution"

Reply:
The main description is at the top of the issue (without a heading, to reduce clutter). The second is "proposed solution" with a remark that suggests to include a prototype/sample code.